_api-reference/ingest-apis/pipeline-failures.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: default
+title: Handling pipeline failures
+parent: Ingest pipelines
+grand_parent: Ingest APIs
+nav_order: 15
+---
+
+## Handling pipeline failures
```

Comment:
@heemin32 Please review the revised failure doc. I tested the examples in Dev Tools console and had no errors. Are there any gaps? Inaccuracies? Thank you.